CONTRIBUTING.md

```diff
@@ -80,6 +80,11 @@ make html
 The above commands build html documentation. Type `make help` for all supported formats,
 type `make clean` to remove the current build files.
 
+When new classes or methods are added, it is recommended to:
+- build html documentation locally,
```

Comment:
Suggest adding the build commands here:
      run:
        python -m pip install --upgrade pip
        pip install -r docs/requirements.txt
        cd docs/
        make html

Reply:
now that's fixed, thanks!